### DISCUSSION TITLE
Is it possible to run intracranial simulations?

### DISCUSSION THREAD

**Discussion by carolina-evangelista** [simnibs/simnibs#229] [8. Simulation workflows] [answered]
Hello, 

I am currently studying the effects of intracranial direct current stimulation in monkeys' auditory cortex. For that, we perform a craniotomy, and the electrode (0,7x0,8x0,2 cm) is placed over the auditory cortex, with a ground electrode attached to the monkey's chamber. Now we are planning to check on how the current spreads and if it is affecting other areas. In this sense, I wonder if there's a way to run this simulation on simNIBS and how I can do that (I already have the monkey's mesh).

Thank you in advance!

**Comment by bjornsig** [comment]
Unfortunately, it's not possible without making significant changes to the code.

**Reply by carolina-evangelista** [reply]
Thank you for your answer! How significant are the changes? 
Are you aware of anyone who has done that? 

Best,
Carolina

**Reply by TorgeW** [reply, marked as answer]
The changes would be significant and SimNIBS doesn't plan to support this in the near future. 

Best :)

**Reply by carolina-evangelista** [reply]
ok, understood. thank you!
